index.js

```diff
@@ -2,61 +2,91 @@
 // Write a JavaScript function that takes a customer's name and account balance and returns a string in the format: "CustomerName has a balance of balance KD."
 function formatCustomerBalance(customerName, balance) {
   // Write code here
+  return `${customerName} has a balance of ${balance} KD`;
 }
 
-formatCustomerBalance("John Doe", 150); // => "John Doe has a balance of 150 KD."
-formatCustomerBalance("Jane Doe", 250); // => "John Doe has a balance of 250 KD."
+console.log(formatCustomerBalance("John Doe", 150)); // => "John Doe has a balance of 150 KD."
+console.log(formatCustomerBalance("Jane Doe", 250)); // => "John Doe has a balance of 250 KD."
 
 // Q2:
 // Write a JavaScript function that takes an array of objects representing books and returns an array of titles published after 2010.
 
 function getBooksPublishedAfter2010(books) {
   // Write code here
+  //if (books.year > 2010) return title;
+  let newArray = [];
+
+  books.filter((book) => {
```

Comment:
you could have used a forEach instead of filter, because here you are not relaying on the return value of the filter, you are just using to loop over the array

Reply:
another way of solving this would be filtering then using map